### ISSUE TITLE
Support Parent WfRunId's in `lhctl`

### ISSUE THREAD

**Issue by coltmcnealy-lh** [littlehorse-enterprises/littlehorse#603]
When `lhctl` was built, a `WfRunId` was just a `string`. Now, we have parent `WfRunId`'s to enable many features such as shared variables, simple foreign keys, etc.

`lhctl` is currently able to _search_ for such `WfRun`'s, but once you are given a `WfRunId`, you cannot retrieve it.

Note that the `WfRunId` struct is used in multiple places, including:
* `TaskRunId`
* `NodeRunId`
* `ExternalEventId`
* `UserTaskRunId`

In order to simplify the command line arguments, we should allow users to use the `_` character to delimit between the parent and child `WfRunId`'s. This is the way the `WfRunIdModel#toString()` and `#fromString()` methods work.

i.e. we should be able to:

```
-> lhct get wfRun 3b64b8d743a8498c84642b8f1dc81d7d_b76adeb2b4f645958cc17ac940e6e8dd

-> lhctl get nodeRun 3b64b8d743a8498c84642b8f1dc81d7d_b76adeb2b4f645958cc17ac940e6e8dd 0 1

-> lhctl get taskRun3b64b8d743a8498c84642b8f1dc81d7d_b76adeb2b4f645958cc17ac940e6e8dd 64f5d7e9f9eb4047bc0865a9b1a5d144
```

**Comment by eduwercamacaro** [comment]
`lhctl run` should be included in this scope.

**Comment by coltmcnealy-lh** [comment]
Good comment. And thanks for taking this ticket! It's important to get before `1.0.0` so I was actually going to do it today, but since you've started, go ahead and finish :smile:

**Comment by coltmcnealy-lh** [comment]
Closed by #604
